ehealthid-rp/src/main/java/com/oviva/ehealthid/relyingparty/ws/ui/Pages.java

```diff
@@ -13,14 +17,30 @@ public Pages(TemplateRenderer renderer) {
     this.renderer = renderer;
   }
 
-  public String selectIdpForm(List<IdpEntry> identityProviders) {
+  public String selectIdpForm(List<IdpEntry> identityProviders, String acceptLanguageHeaderValue) {
```

Comment:
Can you please do the parsing earlier and just pass a locale or list of it? Header parsing has nothing lost here.

Same below.

You could also decorate the `Pages` with a `LocalizedPages` or similar. You should probably pass either a single locale or a list of supported locales, but not a raw header value.

Reply:
Went with opt.1